PostHog/PostHogContext.swift

```diff
@@ -123,8 +123,10 @@ class PostHogContext {
         var properties: [String: Any] = [:]
 
         #if os(iOS) || os(tvOS)
-            properties["$screen_width"] = Float(UIScreen.main.bounds.width)
-            properties["$screen_height"] = Float(UIScreen.main.bounds.height)
+            if let screen = UIApplication.shared.windows.first?.screen {
```

Comment:
This change means that all iOS/tvOS consumers receive a warning unless we're only calling into PostHog from the main thread. If a client's logging system runs off of `main`, accessing `UIApplication.shared` will cause warnings in Xcode. Not sure if that's a priority for y'all, but figured I'd raise.

Reply:
We can check if the captured event is on the main thread and only do the thread jumping if needed to reduce the perf impact as well.
`UIApplication.shared.windows` is also deprecated so the change from A to B didn't make sense IMO.

Reply:
`PostHogReplayIntegration.getCurrentWindow()` is likely the correct way right now, we should move `getCurrentWindow()` out of `PostHogReplayIntegration` and reuse across the SDK, I see more places using `UIApplication.shared` that is just trying to find the active `window`, eg `UIViewController.activeController` and `UIApplication.ph_currentWindow`